src/fetchtastic/cli.py

```
@@ -516,21 +527,39 @@ def run_clean():
+            is_managed_file = item in MANAGED_FILES
+
+            # Only remove managed directories and their contents
+            if is_managed_dir and os.path.isdir(item_path):
+                try:
                     shutil.rmtree(item_path)
-                    print(f"Removed directory: {item_path}")
-            except Exception as e:
-                print(f"Failed to delete {item_path}. Reason: {e}")
-        print(f"Cleaned contents of download directory: {download_dir}")
+                    print(MSG_REMOVED_MANAGED_DIR.format(path=item_path))
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)